packages/cloud_functions/cloud_functions/ios/cloud_functions/Sources/cloud_functions/CodecUtility.swift

```diff
@@ -0,0 +1,117 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+public struct AnyEncodable: Encodable {
+  private let value: Any?
+
+  public init(_ value: Any?) {
+    self.value = value ?? NSNull()
+  }
+
```

Comment:
what is the use of this file? Have we written that ourselves?

Reply:
I added this as the `stream` function in iOS requires an `Encodable` type. Had to create this util class so I could do `var function: Callable<AnyEncodable, StreamResponse<AnyDecodable, AnyDecodable>>`.